api/into_values-freq_agg.md

```diff
@@ -8,7 +8,7 @@ hyperfunction_subfamily: SpaceSavingAggregate
 hyperfunction_type: accessor
 ---
 
-# values()  <tag type="toolkit">Toolkit</tag><tag type="experimental">Experimental</tag>
+# into_values()  <tag type="toolkit">Toolkit</tag><tag type="experimental">Experimental</tag>
 This function returns the data accumulated in a 
 [frequency aggregate][freq_agg].  
```

Comment:
This actually takes any SpaceSavingAggregate, so will work with `topn_agg` or `freq_agg`